### ISSUE TITLE
Google - Accelerated Mobile Pages

### ISSUE THREAD

**Issue by ferdiemmen** [thedaviddias/Front-End-Checklist#157]
Hi @thedaviddias,
I just had another idea for the list. Although I’m not sure what your opinion is.

**AMP**
- https://www.ampproject.org/
- https://developers.google.com/search/docs/guides/about-amp
It is ofcourse a Google-backed project and sites build in AMP even have a special icon in the Google search results.

What do you think? Should it be added as a section?

**Comment by thedaviddias** [comment]
Hi @ferdiemmen, not sure it's a good idea... To be honest, I'm not a real big fan of AMP for websites in general. Maybe some news websites can find an opportunity but not sure that needs to be on a "Front-End Checklist"... I'll be interested to have more POV from the community...

**Comment by elliottregan** [comment]
A lot of the items on this list help you accomplish the same goals as AMP does. That is, small, fast, responsive, etc. I don't think AMP belongs here.

**Comment by dakshshah96** [comment]
AMP is useful for a very small subset of web developers, specifically those working on news/magazine type websites with lots of posts.

I don't think AMP belongs here.

**Comment by amerllica** [comment]
@dakshshah96 , Absolutely you are right, I use AMP inside our mag website, I mean DIGIKALA Mag.
DIGIKALA is the biggest e-commerce of middle east. visit [DIGIKALA Mag](https://www.digikala.com/mag)

**Comment by MrSunshyne** [comment]
AMP is kinda useful for media companies. But i don't think it took off as we expected. 

The main thing lacking is for mobile browsers to **automatically** show and AMP version when available. Even Google Chrome mobile doesn't do that... 

The only app I've seen that favors AMP is the twitter app. If you open an article within twitter, and it has an AMP version, it'll show up automatically, even though the tweet contains the normal link. 


TLDR; Definitely doesn't belong in a "general purpose" front-end checklist. It's only useful for Publishers.

**Comment by thedaviddias** [comment]
@ferdiemmen I think most agree to not mention it. Can you close the ticket pls?

**Comment by ferdiemmen** [comment]
@thedaviddias Yes, and I agree. Thanks for all your opinions and arguments!
